Dockerfile

```diff
@@ -2,9 +2,12 @@ FROM openshift/origin-release:golang-1.10 as builder
 
 WORKDIR /go/src/github.com/aerogear/mobile-security-service
 COPY . .
+RUN curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -
+RUN yum -y install nodejs
 RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh \
 && dep ensure \
-&& go build -o mobile-security-service ./cmd/mobile-security-service/main.go
+&& go build -o mobile-security-service ./cmd/mobile-security-service/main.go \
+&& export STATIC_FILES_DIR=$(CURDIR)/ui/build; ./mobile-security-service -kubeconfig ~/.kube/config
```

Comment:
Folks, it is not working. Any idea how we can solve it?

Reply:
@camilamacedo86 Our Dockerfile should be the same as MDC, at least the parts we used similar to them.

https://github.com/aerogear/mobile-developer-console/blob/master/Dockerfile#L8

Reply:
Thank you it was helpful.